ipv-api/src/main/java/uk/gov/di/authentication/ipv/lambda/ProcessingIdentityHandler.java

```diff
@@ -146,7 +160,17 @@ public APIGatewayProxyResponseEvent handleRequestWithUserContext(
                     "Generating ProcessingIdentityResponse with ProcessingIdentityStatus: {}",
                     processingStatus);
             if (processingStatus == ProcessingIdentityStatus.COMPLETED) {
-                checkAccountInterventionService(pairwiseSubject.getValue(), auditContext);
+                var aisResult =
+                        segmentedFunctionCall(
+                                "AIS: getAccountStatus",
+                                () ->
+                                        accountInterventionService.getAccountStatus(
+                                                internalPairwiseSubjectId, auditContext));
+                if (configurationService.isAccountInterventionServiceActionEnabled()
```

Comment:
Move this flag to within `logoutService.handleAccountInterventionLogout`? To match the pattern in `getAccountStatus`

Reply:
Can also apply this to `AuthenticationCallbackHandler` once ATO-342 is merged

Reply:
I think the return makes that a little tricky?